ql/src/java/org/apache/hadoop/hive/ql/Driver.java

```diff
@@ -1448,7 +1448,7 @@ private void recordValidTxns(HiveTxnManager txnMgr) throws LockException {
     ValidTxnList txnList = txnMgr.getValidTxns();
     String txnStr = txnList.toString();
     conf.set(ValidTxnList.VALID_TXNS_KEY, txnStr);
-    LOG.debug("Encoding valid txns info " + txnStr + " txnid:" + txnMgr.getCurrentTxnId());
+    LOG.info("Encoding valid txns info " + txnStr + " txnid:" + txnMgr.getCurrentTxnId());
```

Comment:
txnStr could be huge in case of large opened txns ..is it safe to add a info log here ?

Reply:
Yeah may be... will change it to debug then.